src/services/sync-checker.ts

```diff
@@ -263,12 +267,12 @@ export class SyncChecker {
 
     // If one or more nodes of this session are not in sync, fire a consensus relay with the same check.
     // This will penalize the out-of-sync nodes and cause them to get slashed for reporting incorrect data.
-    if (syncedNodes.length < 5) {
+    if (syncedNodes.length < nodes.length) {
       const consensusResponse = await pocket.sendRelay(
         syncCheck,
         blockchainID,
         pocketAAT,
-        this.updateConfigurationConsensus(pocketConfiguration),
+        this.updateConfigurationConsensus(pocketConfiguration, nodes.length),
```

Comment:
This should not change the consensus node count. As you are sending a completely new relay and not specifying what node to send it to, PocketJS will use its own interpretation of what the session is and that includes all 5 nodes.